remote/advancedcontainers/set-docker-compose-project-name.md

```diff
@@ -9,9 +9,13 @@ DateApproved: 06/05/2024
 ---
 # Set Docker Compose project name
 
-Visual Studio Code will respect the value of the [COMPOSE_PROJECT_NAME](https://docs.docker.com/compose/reference/envvars/#compose_project_name)  environment variable if set for the VS Code process or in a `.env` file in the root of the project.
+Visual Studio Code will respect the value you configure for the Docker Compose [project name](https://docs.docker.com/compose/project-name/).
 
-For example, after shutting down all VS Code windows, you can start VS Code from the command line as follows:
+You can set the `COMPOSE_PROJECT_NAME` environment variable for the VS Code process, or specify it in a `.env` file in the same folder as the `docker-compose.yml`.
```

Comment:
@ntrogh I would also like to mention the name property, what do you think:

```suggestion
The top-level property `name` in the `docker-compose.yml` can be used to set the project name.

Alternatively, you can set the `COMPOSE_PROJECT_NAME` environment variable for the VS Code process, or specify it in a `.env` file in the same folder as the `docker-compose.yml`.
```

Reply:
I'd like to mention the `name` property in the `docker-compose.yml` because we only recently added support for that on our side and it is probably the easiest way to adjust the project name.

The part about the env variable still makes sense as another way of changing the project name when changing the `name` property is impractical for some reason and because that comes with the additional hurdle of having to restart VS Code.

The env variable use could be considered "advanced" now that we support `name`. We could remove it from the documentation to keep it simple. 🤔

Reply:
Ah, now it makes sense. Your update looks good.